### PR TITLE
Add Ripple LabVIEW Measurement

### PR DESCRIPTION
**Pull request by PJarali-NI**
Added the LabVIEW Ripple Measurement
* This measurement aims to measure the ripples produced in the supply by buck converter.

3 Modes of Operation are supported: -
1) Power ON DUT (To turn on Voltage supplier for DUT)
2) Perform Measurement (To completely measure ripples and give results on readings)
3) Power OFF DUT (To turn off Source and Load for DUT)

Tested on buck converter hardware with following results: -
* Power on DUT
image (35)

* Perform Measurement
image (36)

* Power off DUT
image (37)

### CODE REVIEW

File: source/measurements/ripple/Ripple_PMIC/Get Measurement Details.vi
Comment (ngoliNI) [inline source/measurements/ripple/Ripple_PMIC/Get Measurement Details.vi:1]: Please add 'Annotations' VI and update the measurement details, service class; Refer to the existing measurements.

File: source/measurements/ripple/Ripple_PMIC/Measurement Logic.vi
Comment (ngoliNI) [inline source/measurements/ripple/Ripple_PMIC/Measurement Logic.vi:1]: Don't use 'update results' VI for updating final results

**Review by ngoliNI — CHANGES_REQUESTED** [review]

File: source/measurements/ripple/Ripple_PMIC/Get Measurement Details.vi
Comment (PJarali-NI) [reply source/measurements/ripple/Ripple_PMIC/Get Measurement Details.vi:1]: updated the measurement details vi

File: source/measurements/ripple/Ripple_PMIC/Measurement Logic.vi
Comment (PJarali-NI) [reply source/measurements/ripple/Ripple_PMIC/Measurement Logic.vi:1]: made the changes.

**Pull request closed without merging**